app/controllers/api/v1/packages_controller.rb

```diff
@@ -225,6 +227,31 @@ def send_stock_item_response
         end
       end
 
+      def add_remove_item
+        render nothing: true, status: 204 and return if params[:quantity].to_i.zero?
+        response = Package::Operations.pack_or_unpack(params, User.current_user.id)
+        if response[:success]
+          render json: { packages_inventories: response[:packages_inventory] }, status: 201
+        else
+          render json: { errors: response[:errors] }, status: 422
+        end
+      end
+
+      def fetch_associated_packages
```

Comment:
the term `associated_packages` feels a bit generic. Maybe something like `contained_packages` would be more descriptive 🤔

Reply:
haha, sure @patrixr .